paperqa/types.py

```diff
@@ -375,6 +377,10 @@ def clean_key(cls, value: str) -> str:
     @staticmethod
     def lowercase_doi_and_populate_doc_id(data: dict[str, Any]) -> dict[str, Any]:
         if doi := data.get("doi"):
+            remove_urls = ["https://doi.org/", "http://dx.doi.org/"]
```

Comment:
Can you move this to a module level variable or class variable, with a descriptive name for these two URLs?